src/exo5 - Nested data with traverse/exo5.ts

```diff
@@ -218,13 +235,16 @@ export const performAsyncComputationInSequence: (
 // Use the `sequence` methods from the `option` module to implement the two
 // functions below
 
-export const sequenceOptionTask: (
+export const sequenceOptionTask = (
   optionOfTask: Option<Task<Currency>>,
-) => Task<Option<Currency>> = unimplementedAsync;
+): Task<Option<Currency>> => 
+  option.sequence(task.ApplicativeSeq)(optionOfTask);
+
 
-export const sequenceOptionArray: (
+export const sequenceOptionArray = (
   arrayOfOptions: ReadonlyArray<Option<CountryCode>>,
-) => Option<ReadonlyArray<CountryCode>> = unimplemented;
+): Option<ReadonlyArray<CountryCode>> => 
+  readonlyArray.sequence(option.Applicative)(arrayOfOptions);
```

Comment:
here you could use the more concise `option.sequenceArray` which converts `ReadonlyArray<Option<T>>` to `ReadonlyArray<Array<T>>`